src/utils/slugModifiers/isSlugLegacyOrEYFS.ts

```diff
@@ -0,0 +1,8 @@
+import isSlugEYFS from "./isSlugEYFS";
+import isSlugLegacy from "./isSlugLegacy";
+
+const isSlugLegacyOrEYFS = (slug: string) => {
```

Comment:
sorry this is minor but I think a more usage based name would work well here, like `shouldUseLegacyApi`, so we don't need the consumers of the function to know exactly what this method is doing